native/packages/react-native-bpk-component-pagination-dots/src/BpkPaginationDots-test.common.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import BpkPaginationDots from './BpkPaginationDots';
+
+const commonTests = () => {
+  describe('BpkPaginationDots', () => {
+    [...Array(6)].forEach((_, index) => {
```

Comment:
I have a hunch that it might be worth going to 10 or something, just because I feel like there is a lot of intricate logic and we could break something that only affects the `>5` scenario at higher values

Reply:
There are several scenarios that aren't being captured between the tests, such as `pageCount={4} selectedIndex={3}`

Could we nest these, and do something like:
```
  describe('BpkPaginationDots', () => {
    [...Array(7)].forEach((_, index1) => {
      [...Array(7)].forEach((_, index2) => {
        it(`should render correctly when pageCount is ${index + 1}`, () => {
          const tree = renderer
            .create(
              <BpkPaginationDots
                accessibilityLabel={`0 of ${index + 1}`}
                pageCount={index1 + 1}
                selectedPageIndex={index2}
              />,
            )
            .toJSON();
          expect(tree).toMatchSnapshot();
        });
      });
    });
```

Reply:
Actually, screw that! Travis takes too flippin' long to run. Just do `expect(true)` and I'll be happy 😆

Reply:
Nah I'm doing it!